src/components/dashboard/main-content/course-enrollments/course-cards/styles/_CourseCard.scss

```diff
@@ -44,4 +45,14 @@
       display: block;
     }
   }
+
+  .custom-course-card-badge {
+    line-height: 1.75;
+    width: fit-content;
+    height: 30px;
+  }
```

Comment:
These `Badge`'s should be the standard `Badge` components from Paragon; no need for these custom `Badge` overrides. The `Badge` components should look like they do on the Paragon docs site when running the `@edx/brand-edx.org` theme.